src/adapters/Authentication/index.ts

```diff
@@ -0,0 +1,11 @@
+import { post } from 'adapters/Base';
+import { Config } from 'config';
+
+export const signIn = (email: string, password: string) =>
```

Comment:
do we have any validation here or submit the function below? Or are you gonna add it in the next PR?

Reply:
@nvminhtue we don't have the validation here, I will add it into the integrate task 🙏